src/Microsoft.IdentityModel.Tokens/EventBasedLRUCache.cs

```diff
@@ -226,7 +186,7 @@ internal int RemoveExpiredValues()
                     if (node.Value.ExpirationTime < DateTime.UtcNow)
                     {
                         _doubleLinkedList.Remove(node);
-                        if (_map.TryRemove(node.Value.Key, out var cacheItem))
+                        if (base.TryRemoveInternal(node.Value.Key, out var cacheItem))
```

Comment:
what is the reason for using 'base.'?

Reply:
Initially it was used for me to track all the places calling the new base class methods, I forgot to remove them after I finished.